client/src/pages/DirectoryPage/DirectoryGroups.js

```diff
@@ -14,223 +14,196 @@ import picture1 from "../../assets/images/card_small1.png";
-            icon={"white"}
-            onClick={pillClick}
-            selectId={0}
-          />
-          <Pill
-            className="Pill"
-            text={"Traveling"}
-            size={"wide"}
-            color={"pink"}
-            icon={"white"}
-            onClick={pillClick}
-            selectId={0}
-          />
-          <Pill
-            className="Pill"
-            text={"Traveling"}
-            size={"wide"}
-            color={"pink"}
-            icon={"white"}
-            onClick={pillClick}
-            selectId={0}
-          />
-          <Pill
-            className="Pill"
-            text={"Traveling"}
-            size={"wide"}
-            color={"pink"}
-            icon={"white"}
-            onClick={pillClick}
-            selectId={0}
-          />{" "}
-          <Pill
-            className="Pill"
-            text={"Traveling"}
-            size={"wide"}
-            color={"pink"}
-            icon={"white"}
-            onClick={pillClick}
-            selectId={0}
-          />{" "}
-          <Pill
-            className="Pill"
-            text={"Traveling"}
-            size={"wide"}
-            color={"pink"}
-            icon={"white"}
-            onClick={pillClick}
-            selectId={0}
-          />
-          <Pill
-            className="Pill"
-            text={"Traveling"}
-            size={"wide"}
-            color={"pink"}
-            icon={"white"}
-            onClick={pillClick}
-            selectId={0}
-          />
-          <Pill
-            className="Pill"
-            text={"Traveling"}
-            size={"wide"}
-            color={"pink"}
-            icon={"white"}
-            onClick={pillClick}
-            selectId={0}
-          />
-          <Pill
-            className="Pill"
-            text={"Traveling"}
-            size={"wide"}
-            color={"pink"}
-            icon={"white"}
-            onClick={pillClick}
-            selectId={0}
+  componentDidMount() {
+    axios.get(`${BASE_URL}/groups`)
+      .then(response => {
+        this.setState({ 
+          groups: response.data
+        })
+      })
+      .catch(error => {
+        console.log('Error while fetching and parsing API request', error);
+      })
+  }
+
+  render() {
+    console.log(this.state)
+    const groups = this.state.groups ? this.state.groups : [];
+    return (
+      <div className="DirectoryPage">
+        <NavBar page={0} profileImage={people} />
+        <Banner background={Banner__Directory}>
+          <h3 style={{ margin: 0 }}>Directory: My Groups</h3>
+          <InputTextField
+            type="text"
+            variation="wide"
+            name="search directory"
+            placeholder="Search Groups"
           />
-        </div>{" "}
-        <a className="seeMore-Button">See More</a>
-        <div>
-          <header className="CommunityPage__header">Groups in WASHINGTON, DC:</header>
+        </Banner>
+        <div className="secondNav">
```

Comment:
Can you update your classes to follow the naming convention outlined in the readme?